pkg/apiserver/storage/file/file.go

```diff
@@ -306,6 +307,13 @@ func (s *Storage) Watch(ctx context.Context, key string, opts storage.ListOption
 // match 'opts.ResourceVersion' according 'opts.ResourceVersionMatch'.
 func (s *Storage) Get(ctx context.Context, key string, opts storage.GetOptions, objPtr runtime.Object) error {
 	filename := s.filePath(key)
+
+	// Since it's a get, check if the dir exists and return early as needed
+	dirname := filepath.Dir(filename)
```

Comment:
should this be `dirname := s.dirPath(key)` just to keep things consistent with list? it seems like little differences can make debugging confusing if one or the other stops working

Reply:
Thanks! Yeah we don't want bugs on different OSes. I now have a windows VM if we ever need to test something over there.